README.adoc

```diff
@@ -0,0 +1,110 @@
+[[landrush-dns-for-vagrant-build-status-build-status]]
+= Landrush: DNS for Vagrant
+https://travis-ci.org/vagrant-landrush/landrush[image:https://travis-ci.org/vagrant-landrush/landrush.png[Build
+Status]]
+https://ci.appveyor.com/project/hferentschik/landrush-3agrx/branch/master[image:https://ci.appveyor.com/api/projects/status/3iv8sv5v73s15mt6/branch/master?svg=true[Build
+status]]
+
+:toc:
+:toc-placement!:
```

Comment:
This does not seem to render. At least not if I view it as part of the pull request.